lib/metasploit_payloads/mettle.rb

```diff
@@ -176,8 +176,21 @@ def self.available_extensions(platform)
       end
 
       extensions = ::Dir.entries(dir_path)
+      extensions.select! { |extension| !extension.end_with?('.bin') }
+      # The stdapi extension is baked in
+      extensions << 'stdapi'
       # Only return extensions!
-      extensions - [ '.', '..', 'mettle', 'mettle.bin' ]
+      extensions -= [ '.', '..', 'mettle', 'mettle' ]
```

Comment:
Did you mean to type `mettle` twice here? This almost seems like a case where using something like `.uniq!` might make more sense instead of doing this.

Reply:
Its also possible this was meant to be `mettle.bin` and you left this in after adding the line `extensions.select! { |extension| !extension.end_with?('.bin') }` which would exclude the `mettle.bin` case due to the name ending with `.bin`?

Reply:
Yeah the extra word is redundant. Looks like I dropped the `bin` suffix but the whole word should have been removed.

Reply:
Resolved with https://github.com/rapid7/mettle/pull/215/commits/b6e66bf7179e5ca9cf0b00aa8cf5e573361dcb56